### DISCUSSION TITLE
Problem deploying Backpack in my AWS integration server

### DISCUSSION THREAD

**Discussion by alldefidev** [Laravel-Backpack/community-forum#836] [Q&A (Help)]
Hello, I have installed BACKPACK 6 PRO on my development server, and it's working correctly. I am trying to deploy it on the integration server on AWS, and I'm having issues loading styles and encountering a bunch of errors in the console. I have checked that my .env file has the correct domain. 
The domain is a subdomain of the one I have licensed, so that shouldn't be the issue either.
Attached is a screenshot with the errors visible in the console. 
Thank you very much for your help!
![Captura de pantalla 2024-02-11 a las 18 45 11](https://github.com/Laravel-Backpack/community-forum/assets/159048855/3502e3d2-5a38-4999-8892-bd068a6ee300)

**Comment by jcastroa87** [comment]
Hi @alldefidev did you try with:

```
php artisan basset:check
```

Or check [basset docs](https://github.com/Laravel-Backpack/basset) to clear the cache or check the FAQ when deploying on VPS.

Cheers.

**Comment by alldefidev** [comment]
If I imake a manul isntallation after deploy php artisan backpack:install using command line in server, everythoing works fine.
I tried to add         
"post-composer-update": [
            "php artisan backpack:install --no-interaction"
        ]
but it doesn´t work.

**Reply by pxpm** [reply]
@alldefidev on deploy, you should rebuild the cache, maybe that's the step that's missing:

```bash
# fetch the repo etc
# run your deploy scripts migrations etc
# at the end of the deploy, add the follwing:
php artisan basset:clear && php artisan basset:cache
```
If it still doesn't work, please after do a `php artisan basset:check` like @jcastroa87 suggested and let me know the results. 

Cheers

**Comment by jcastroa87** [comment]
Due to not activity i will close this issue, but please feel free to re-open or create a new one if needed.

Cheers.
